### PR TITLE
bpo-41861: Add NEWS and normalise _sqlite3 Cache/Node type struct naming

### PR DESCRIPTION
**Pull request by erlend-aasland**
https://bugs.python.org/issue41861

### CODE REVIEW

**Comment by erlend-aasland** [comment]
@corona10 Would you mind taking a quick look at this? Just closing bpo-41861 by adding a NEWS entry and normalising Cache/Node struct names (so that the naming is consistent with the other type structs in `_sqlite3`.)

(I closed #22419, since it went in limbo.)

**Review by corona10 — COMMENTED** [review]
@erlend-aasland 

Can you provide a discussion about normalizing?
we often do not accept this kind of PR ;)

**Comment by erlend-aasland** [comment]
> Can you provide a discussion about normalizing?
> we often do not accept this kind of PR ;)

Yes, I'm aware of that, but since this is the final PR of bpo-41861, I thought an exception would be acceptable, since the other PR's relating to this bpo have used this naming scheme ;) Please? :)

File: Misc/NEWS.d/next/Library/2020-10-01-15-44-52.bpo-41861.YTqJ7z.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Convert :mod:`sqlite3` to use heap types (PEP 384.)
```
Comment (vstinner) [inline Misc/NEWS.d/next/Library/2020-10-01-15-44-52.bpo-41861.YTqJ7z.rst]:
```suggestion
Convert :mod:`sqlite3` to use heap types (PEP 384).
```

Comment (erlend-aasland) [reply Misc/NEWS.d/next/Library/2020-10-01-15-44-52.bpo-41861.YTqJ7z.rst]: Thanks, fixed.

**Comment by vstinner** [comment]
Merged. I rewrote your commit message to make it shorter.

**Pull request merged**